### PR TITLE
MK3 sync Prusa-Error-Codes

### PR DESCRIPTION
**Pull request by 3d-gussner**
Added script `check-pce.sh` to check if MK3 MMU messages are same as [PCE MMU](https://github.com/prusa3d/Prusa-Error-Codes/blob/master/04_MMU/error-codes.yaml) 

Please also review https://github.com/prusa3d/Prusa-Error-Codes/pull/93

@DRracer The error code `04900` is in PCE but not in MK3 branch

### CODE REVIEW

File: lang/po/Firmware_cs.po [hunk 1]
```diff
@@ -2470,7 +2470,7 @@ msgstr ""
 
 #. MSG_TITLE_MCU_UNDERVOLTAGE_VCC c=20
 #: ../../Firmware/mmu2/errors_list.h:166 ../../Firmware/mmu2/errors_list.h:211
-msgid "MCU UNDERVOLTAGE VCC"
+msgid "MMU MCU UNDERPOWER"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po:2473]: we may need to update the translations in this case

**Review by DRracer — APPROVED** [review]
nice :+1: , well structured

File: lang/po/Firmware_cs.po (hunk 1 above)
Comment (3d-gussner) [reply lang/po/Firmware_cs.po:2473]: Working on this.

Comment (3d-gussner) [reply lang/po/Firmware_cs.po:2473]: BTW Undervoltage != underpower

Voltage != power

**Comment by 3d-gussner** [comment]
@DRracer Please review newest changes.

**Review by DRracer — APPROVED** [review]
well done :+1:

**Pull request merged**